### PR TITLE
Un-Disabling preview 2 supported features

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 1]
```diff
@@ -899,16 +899,16 @@ static void validateMultipageReceiptData(List<RecognizedForm> recognizedReceipts
             .getValue().asString());
         assertEquals(EXPECTED_MULTIPAGE_PHONE_NUMBER_VALUE, receiptPage1Fields.get("MerchantPhoneNumber")
             .getValue().asPhoneNumber());
-        assertNotNull(receiptPage1Fields.get("Total").getValue().asFloat());
+        // service issue total returned as null?
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:902]: do you think creating a github issue and putting it here will help to keep track of this?

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 2]
```diff
@@ -899,16 +899,16 @@ static void validateMultipageReceiptData(List<RecognizedForm> recognizedReceipts
...
+        // assertNotNull(receiptPage1Fields.get("Total").getValue().asFloat());
         assertNotNull(receiptPage1.getPages());
         assertEquals(ITEMIZED_RECEIPT_VALUE, receiptPage1Fields.get("ReceiptType").getValue().asString());
 
-        // Assert no fields, tables and lines on second page
+        // Assert no fields and lines on second page
         assertEquals(0, receiptPage2.getFields().size());
         List<FormPage> receipt2Pages = receiptPage2.getPages();
         assertEquals(1, receipt2Pages.size());
-        assertEquals(0, receipt2Pages.stream().findFirst().get().getTables().size());
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:910]: ohh interesting. is it not mandatory in the swagger? or what changed? a regression?

Comment (maririos) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:910]: ohh yeah I saw this too. NM

**Review by maririos — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java [hunk 3]
```diff
@@ -456,7 +456,9 @@ private static float setDefaultConfidenceValue(Float confidence) {
             .map(fieldValue -> {
                 FieldData valueData = null;
                 // ARRAY has ho value data, such as bounding box.
-                if (ARRAY != fieldValue.getType()) {
+                if (ARRAY != fieldValue.getType()
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java:459]: what happens if the field type is not ARRAY only?

Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java:459]: They will have valueData except for the mentioned two cases.

**Review by mssfang — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java (hunk 1 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java]: https://github.com/Azure/azure-sdk-for-java/issues/17661

**Pull request merged**